stackslib/src/burnchains/tests/mod.rs

```diff
@@ -899,13 +899,10 @@ fn verify_keys_accepted(node: &mut TestBurnchainNode, prev_keys: &[LeaderKeyRegi
         assert!(tx_opt.is_some());
 
         let tx = tx_opt.unwrap();
-        match tx {
-            BlockstackOperationType::LeaderKeyRegister(ref op) => {
-                assert_eq!(*op, *key);
-            }
-            _ => {
-                assert!(false);
-            }
+        if let BlockstackOperationType::LeaderKeyRegister(op) = tx {
+            assert_eq!(op, *key);
+        } else {
+            panic!();
```

Comment:
Not really necessary, but I prefer to always put some text in a `panic!()`:

 ```suggestion
             panic!("Expected key registration tx!");
```

Reply:
Also a `let else` would be *slightly* less text and indentation than `if let` here, but it really doesn't matter